package.json

```diff
@@ -22,6 +22,7 @@
     "@gera2ld/plaid": "~1.4.12",
     "@gera2ld/plaid-vue": "~1.4.9",
     "@gera2ld/plaid-webpack": "~1.5.0",
+    "@types/chrome": "latest",
```

Comment:
`yarn.lock` should be updated.

Reply:
Done.
P.S. turns out my IDE was using npm by default so I've switched it to yarn now.